### PR TITLE
Implementing stat, Statistics box styling, about page

### PR DESCRIPTION
**Pull request by lRolwes**
Problem: 
- About Page empty
- Stats difficult to read
- Not all stats implemented
- Code for getStatistics function difficult to read in PostgresConnector.py

Files Changed:
- AboutPage.js
- ExploreSystems.js
- App.css
- PostgresConnector.py

What Changed:
- About page filled with 4 sections
- Styling on filters and statistics changed to improve readability
- last implementable stat implemented
- Code in PostgresConnector.py changed for better organization/readability

Pics:
image
image

### CODE REVIEW

File: Backend/PostgresConnector.py [hunk 1]
```diff
@@ -63,27 +63,25 @@ def getStatistics(self,filters):
...
-        sql = "SELECT SUM(is_Newton::int) FROM public.data" + whereText 
-        cur = self.connection.cursor()
-        cur.execute(sql)
-        result3 = cur.fetchall()
+        height = cur.fetchall()
         #Average Resultant
         sql = "SELECT AVG( models_original_resultant) FROM public.data" + whereText 
         cur.execute(sql)
-        result4 = cur.fetchall()
+        resultant = cur.fetchall()
```
Comment (Evan-Richmond) [inline Backend/PostgresConnector.py:82]: thanks for fixing naming conventions here, much more clear

File: Frontend/src/pages/AboutPage.js [hunk 2]
```diff
@@ -2,13 +2,118 @@ import React from 'react';
...
+    <p>
+        <a href="https://docs.docker.com/desktop/install/windows-install/" target="_blank">Docker Desktop Installation Guide</a>, 
+        or use the appropriate package manager on a UNIX based system.
+    </p>
+    <p>
+        (Homebrew will likely be the best for Mac, but research your Linux distro to use the package manager it was installed with.
+        The command will likely look like this: <code>sudo PACKAGE_MANAGER_NAME install docker</code>)
+    </p>
+
+    <h3>Setup Docker Container</h3>
```
Comment (Evan-Richmond) [inline Frontend/src/pages/AboutPage.js:71]: These setup instructions are great, should be easy to replace when we change our database setup

**Review by Evan-Richmond — APPROVED** [review]
Looks great!

**Review by AJelliss — APPROVED** [review]
Looks Amazing! can't wait to see it merged

**Review by AbhilashKotha — CHANGES_REQUESTED** [review]
@lRolwes The Home new styles are amazing. Information on the Home page looks great! I just have a few more improvement suggestions. 

1. the statistics div goes to the bottom on my computer.
image
image

2. The Home page has a lot of technical information. Let us assume that not all of our users have technical knowledge and will use the app with a URL after it is hosted. For now, you can just redirect them to a current .md file or you can also create new files for troubleshooting steps. 

so my suggestion would be to move eveything after Github URL to github and add those redirection URLs in the website
3. Increase the font size and font weight for statistics just a little bit.

**Review by AbhilashKotha — APPROVED** [review]

**Pull request merged**